src/components/EquipmentList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import EquipmentExpansionList from "./EquipmentExpansionList";
+import Equipment from "../resources/Equipment";
+import EquipmentStandardList from "./EquipmentStandardList";
+
+const useStyles = makeStyles({
+  root: {
+    width: "100%",
+  },
+});
+
+interface EquipmentListProps {
+  equipmentList: Equipment[] | undefined;
```

Comment:
Can we use a default `[]`?  Any reason not to?
```suggestion
  equipmentList: Equipment[];
```

Reply:
I believe that this was because when querying you can query without any results which is undefined. I'll check again

Reply:
The equipment[] -> filteredEquipment[] should always return [].  A query shouldn't make it returned undefined.